src/main/java/com/minecraftabnormals/upgrade_aquatic/core/events/EntityEvents.java

```diff
@@ -146,6 +148,17 @@ public static void onEntityUpdate(LivingUpdateEvent event) {
 				}
 			}
 		}
+		if (entity instanceof DrownedEntity) {
+			Pose pose = EntityEvents.getHorizontalMotion(((DrownedEntity) entity).getMotion()) >= 0.025F && ((DrownedEntity) entity).getEntityWorld().getFluidState(((DrownedEntity) entity).getPosition().down()).isTagged(FluidTags.WATER) ? Pose.SWIMMING : Pose.STANDING;
```

Comment:
There is a lot of unnecessary casting here.
One case being `Entity` has `getMotion()`, so there's no need to cast to `DrownedEntity`.